configs/components/openssl.rb

```diff
@@ -11,8 +11,9 @@
     pkg.build_requires "pl-gcc-#{platform.architecture}"
     pkg.build_requires 'runtime' if platform.is_huaweios?
     # needed for the makedepend command
+    pkg.build_requires 'imake' if platform.name =~ /^el/
```

Comment:
That's a good question. I think something about customizing the target arch for cross-compiles invokes makedepend in a way that native builds do not. But I haven't dug into that. 

Either way, without this the build clearly fails with an error about needing the makedepend command, and succeeds when it has it. Distros are all over the map in how they package it.